src/TLuaInterpreter.cpp

```diff
@@ -3203,92 +3203,212 @@ int TLuaInterpreter::hideUserWindow(lua_State* L)
     return 0;
 }
 
+// No documentation available in wiki - internal function
+void TLuaInterpreter::setBorderSize(lua_State* L, int size, int position, bool updateEvent)
+{
+    Host& host = getHostFromLua(L);
+    // position: 0 = top, 1 = right, 2 = bottom, 3 = left
```

Comment:
So we don't have to use a command explaining numbers with a special meaning, enums work well here.

Reply:
Do I need to include those first or where do they arrive from?

Reply:
Should already be available, just use them as-is.